app/web/features/communities/events/EventCard.tsx

```diff
@@ -155,41 +111,41 @@ export default function EventCard({ event, className }: EventCardProps) {
               />
             )}
           </CardMedia>
+
           <CardContent>
-            <Typography component="h3" className={classes.title}>
+            <Typography
+              variant="body2"
+              color="textSecondary"
+              className={classes.eventTime}
+              gutterBottom
+              /* title useful to hover in case it's too long for the card */
+              title={formattedEventDates}
+            >
+              {formattedEventDates}
+            </Typography>
+
+            <Typography variant="h3" gutterBottom className={classes.title}>
               {event.title}
             </Typography>
-            <Typography className={classes.subtitle} noWrap variant="body2">
+            <Typography noWrap variant="body2" gutterBottom>
               {event.offlineInformation
                 ? event.offlineInformation.address
                 : t("communities:virtual_event_location_placeholder")}
             </Typography>
-            <ul className={classes.detailsList}>
-              <li>
-                <CalendarIcon className={classes.icon} />
-                <Typography variant="body1" className={classes.eventTime}>
-                  {`${startTime.format("LLL")} - ${endTime.format(
-                    endTime.isSame(startTime, "day") ? "LT" : "LLL"
-                  )}`}
-                </Typography>
-              </li>
-              <li>
-                <AttendeesIcon className={classes.icon} />
-                <Typography variant="body1">
-                  {t("communities:attendees_count", {
-                    count: event.goingCount + event.maybeCount,
-                  })}
-                </Typography>
-              </li>
-            </ul>
-            <div className={classes.otherInfoSection}>
-              <Typography variant="h4">
-                {t("communities:details_subheading")}
-              </Typography>
-              <Typography className={classes.content} variant="body1">
+
+            <Divider spacing={1} />
+
+            <Box>
```

Comment:
```suggestion
            <div>
```
Based on reasons mentioned in previous comment, especially when we don't need it for additional styling rules other than grouping child elements for flex/grid etc